src/config/rcxml.c

```diff
@@ -1059,6 +1106,14 @@ rcxml_finish(void)
 		zfree(field);
 	}
 
+	struct window_rule *rule, *rule_tmp;
+	wl_list_for_each_safe(rule, rule_tmp, &rc.window_rules, link) {
+		wl_list_remove(&rule->link);
+		zfree(rule->app_id);
+		action_list_free(&rule->actions);
+		zfree(rule);
+	}
+
```

Comment:
I think the current globals aren't reset, see the section below.

Reply:
Thanks. Sorted.